packages/venia-ui/lib/components/ProductOptions/option.js

```diff
@@ -38,14 +38,14 @@ const Option = props => {
     const [selection, setSelection] = useState(null);
 
     const initialSelection = useMemo(() => {
-        let selection = {};
-        if (selectedValue) {
-            selection =
-                values.find(value => value.default_label === selectedValue) ||
-                {};
+        let initialSelection = {};
+        const searchValue = selection || selectedValue;
+        if (searchValue) {
+            initialSelection =
+                values.find(value => value.default_label === searchValue) || {};
```

Comment:
This is the coupling that basically enforces we use `label` as the indicator of selection. Not a big deal, but wanted to point this out.

Reply:
But only for product options, right? I don't see anything wrong with it; is there something you would like to use instead?

Reply:
I just think the normalization should occur elsewhere.